MVCapplication/src/main/resources/message.properties

```diff
@@ -0,0 +1,4 @@
+registration.errorMessage=errorMessage
+
+
+login.informationMessage=informationMessage
```

Comment:
this can be in java, but real messages which will be shown for user have to be here, for easy localization support in future